lstchain/image/muon/muon_analysis.py

```diff
@@ -271,4 +272,23 @@ def tag_muon(image,thr_low=0,thr_up=1.e10):
     `bool` it determines whether a muon was tagged or not
 
     """
-    return image.sum() > thr_low and image.sum() < thr_up
\ No newline at end of file
+    return image.sum() > thr_low and image.sum() < thr_up
+
+def tag_muon_pix(image,thr_low=50,thr_up=300):
+    """
+    Tag muon with a double threshold on the number of pixels above 10 photoelectrons 
+    Default values apply elimination of pedestal and calibration events
+
+    Paramenters
+    ---------
+    image:      `np.ndarray` number of photoelectrons in each pixel
+    thr_low: `int` lower threshold for number of pixel > 10 pe  
+    thr_up: `int` upper threshold for number of pixel > 10 pe
+
+    Returns
+    ---------
+    `bool` it determines whether a muon was tagged or not
+
+    """
+
+    return ((np.size(image[0][image[0]>10.]) < thr_up) and (np.size(image[0][image[0]>10.]) > thr_low))
```

Comment:
I would suggest to make the phe_threshold per pixel (10 in this case) another variable to be set on the function

Reply:
I would use the tag_muon instead of the cut in size, since from a quick analysis the tagging using the nr of pe, is not so performing as the one in size.